Orange/canvas/__main__.py

```diff
@@ -593,12 +723,10 @@ def show_message(message):
                  open_requests[-1])
         canvas_window.load_scheme(open_requests[-1].toLocalFile())
 
-    # initialize notifications
-    setup_notifications()
-
     # local references prevent destruction
-    update_check = check_for_updates()
+    update_check = check_for_updates(notif_server)
```

Comment:
I was trying to avoid using a singleton pattern for the NotificationServer, but perhaps it's for the best – being able to call a class method to register notifications (`NotificationServer.registerNotification(notif: Notification)`) is much simpler than having to link back to the server instance in CanvasMainWindow.